tests/jquery.flot.Test.js

```diff
@@ -462,6 +462,44 @@ describe('flot', function() {
         });
     });
 
+    describe('findNearbyInterpolationPoint', function() {
+        var placeholder, plot, sampledata = [[0, 1], [1, 1.1], [2, 1.2]];
```

Comment:
Could you please add 2 more tests: one for an empty sampledata and another one with a single point.